api service integration changes (#5958)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/build-api-integration/main/index.md

### Register your API service integration

The OIN Wizard provides a mechanism to register and install your integration in your Okta org (similar to how your customer admins would install your integration) and obtain the credentials to test your Client Credentials flow. Your test integration is available only on your Okta org during the test.

> **Note:** From the OIN Wizard test option, the test install and authorize pages are the same pages that your customers experience when your service integration is in the OIN. After you publish your integration in the OIN catalog, it appears in the **Applications** > **API Service Integrations** page of your customer's Admin Console.

To register your API service registration, see [Submit an integration with the OIN Wizard](https://developer.okta.com/docs/guides/submit-oin-app/openidconnect/main/).

### Authorize a test integration

Use the test functionality in the OIN Wizard to obtain test integration credentials:

> 1. From the OIN Wizard, click **Applications** > **Your OIN Integrations** and click the **More actions ⋮** icon and click **Test integration**.

Reviewer: ```suggestion
1. In the OIN Wizard, go to **Applications** > **Your OIN Integrations**.
1. Click the **More actions ⋮** icon and select **Test integration**.
```

Author: it is the vertical ellipsis

To edit your API service integration submission fields:

1. From the OIN Wizard, click **Applications** > **Your OIN integrations**.

> 1. Click the integration that you want to open from the list. Click **Edit** next to your integration submission.

Reviewer: ```suggestion
1. Select an integration from the list. Click **Edit** next to your integration submission.
```
